### ISSUE TITLE
FNX3-17091 ⁃ InflateException TabCounter: Binary XML file line #18: Error inflating class ImageView

### ISSUE THREAD

**Issue by ghost** [mozilla-mobile/fenix#3538]
https://sentry.prod.mozaws.net/operations/fenix-nightly/issues/5950597/

This is primarily happening on Android 5 - is this an invalid SVG file that we can fix? Or revert to an image?

Are there other SVG files that could be problematic?

Looks like this is the same RuntimeException reported as:

https://sentry.prod.mozaws.net/operations/fenix-nightly/issues/5954445

```
java.lang.NumberFormatException: Invalid float: "-1.1.9"
    at java.lang.StringToReal.invalidReal(StringToReal.java:63)
    at java.lang.StringToReal.initialParse(StringToReal.java:160)
    at java.lang.StringToReal.parseFloat(StringToReal.java:323)
    at java.lang.Float.parseFloat(Float.java:306)
    at android.util.PathParser.getFloats(PathParser.java:178)
    at android.util.PathParser.createNodesFromPathData(PathParser.java:59)
    at android.graphics.drawable.VectorDrawable$VFullPath.updateStateFromTypedArray(VectorDrawable.java:1439)
    at android.graphics.drawable.VectorDrawable$VFullPath.inflate(VectorDrawable.java:1421)
    at android.graphics.drawable.VectorDrawable.inflateInternal(VectorDrawable.java:542)
    at android.graphics.drawable.VectorDrawable.inflate(VectorDrawable.java:458)
    at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:1100)
    at android.graphics.drawable.Drawable.createFromXml(Drawable.java:1022)
    at android.content.res.Resources.loadDrawableForCookie(Resources.java:2682)
    at android.content.res.Resources.loadDrawable(Resources.java:2578)
    at android.content.res.TypedArray.getDrawable(TypedArray.java:749)
    at android.widget.ImageView.<init>(ImageView.java:146)
    at android.widget.ImageView.<init>(ImageView.java:135)
    at androidx.appcompat.widget.AppCompatImageView.<init>(AppCompatImageView.java:3)
    at androidx.appcompat.app.AppCompatViewInflater.createImageView(AppCompatViewInflater.java:2)
    at androidx.appcompat.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:37)
    at androidx.appcompat.app.AppCompatDelegateImpl.onCreateView(AppCompatDelegateImpl.java:22)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:725)
    at android.view.LayoutInflater.rInflate(LayoutInflater.java:806)
    at android.view.LayoutInflater.rInflate(LayoutInflater.java:809)
    at android.view.LayoutInflater.inflate(LayoutInflater.java:479)
    at android.view.LayoutInflater.inflate(LayoutInflater.java:414)
    at android.view.LayoutInflater.inflate(LayoutInflater.java:365)
    at org.mozilla.fenix.components.toolbar.TabCounter.<init>(TabCounter.kt:4)
```

**Comment by sarentz** [comment]
@NotWoods This looks similar https://sentry.prod.mozaws.net/operations/fenix-nightly/issues/5954008 - it is for a different asset but it fails on the same `Invalid float: "-1.1.9"`

**Comment by abodea** [comment]
Verified as fixed on the latest RC build **`1.0.0-rc.2(Build #11700147 68.0.20190611143747) 6/19.`** with **Samsung Galaxy Tab(Android 5.1.1).**
As this issue refers to **`TabCounter`** there was no problems/crashes regarding tabs or any menu/section from Fenix.
